### PR TITLE
[CP] Additional upgrade instructions for Katello 4.0/Foreman 2.4.

### PR DESCRIPTION
**Pull request by jjeffers**
Cherry-pick into:

* [ ]  Foreman 2.4 (Satellite 6.9)

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc [hunk 1]
```diff
@@ -138,3 +138,15 @@ In such a case, change directory, for example to the *_root_* user's home direct
 ----
 # {package-install-project} tfm-rubygem-smart_proxy_discovery.noarch
 ----
+. Remove all Pulp2 content on each smart proxy.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc:141]: Will Sat customers have to do this step in time or will it be upstream-only?

**Review by melcorr — COMMENTED** [review]
One question for you, @jjeffers

File: guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc (hunk 1 above)
Comment (jjeffers) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc:141]: I beleive it's for both upstream and down, but I will check.

Comment (jjeffers) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc]: @melcorr For now there is no difference between upstream and down.

File: guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc [hunk 2]
```diff
@@ -131,3 +131,16 @@ endif::[]
...
 # {foreman-maintain} service restart
 ----
+. Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
+. If you make changes in the previous step, restart the `{foreman-maintain}` services.
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} service restart
+----
+. If you have migrated content from Pulp2 to Pulp3, remove all old (Pulp2) content.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc:141]:
```suggestion
. If you have migrated content from Pulp 2 to Pulp 3, remove all Pulp 2 content.
```

Sorry just saw this now. Just to be consistent everywhere.

**Review by melcorr — CHANGES_REQUESTED** [review]
Thanks @jjeffers 
After this small change, we are good to go!

File: guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc [hunk 3]
```diff
@@ -138,7 +138,7 @@ endif::[]
 ----
 # {foreman-maintain} service restart
 ----
-. If you have migrated content from Pulp2 to Pulp3, remove all old (Pulp2) content.
+. If you have migrated content from Pulp2 to Pulp3, remove all Pulp2 content.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc:141]: @jjeffers  can we please have a space between Pulp and the versions also? For example Pulp 2 and Pulp 3? 
Sorry to nitpick at you so much.

Comment (jjeffers) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc:141]: Ooh! I missed that earlier. No worries, I am grateful for the reviews.

**Review by melcorr — APPROVED** [review]
Thanks. 
@jjeffers  if you don't need other reviews from the Katello folks, I am happy to merge this!

**Comment by jjeffers** [comment]
@jlsherrill can you take a look and comment on this update? Thanks!

**Review by jlsherrill — APPROVED** [review]
LGTM!

**Pull request merged**